packages/react-core/src/components/Hint/examples/HintBasicWithoutTitle.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {
+  Hint,
+  HintBody,
+  HintFooter,
+  Button,
+  Dropdown,
+  DropdownItem,
+  DropdownSeparator,
+  KebabToggle
+} from '@patternfly/react-core';
+
+export const BasicHint: React.FunctionComponent = () => {
```

Comment:
Similar nitpick to the previous comment re: the exported component name matching the file name